packages/core/src/internal/runSaga.js

```diff
@@ -47,18 +47,27 @@ export function runSaga(options, saga, ...args) {
     effectMiddlewares.forEach(effectMiddleware => check(effectMiddleware, is.func, MIDDLEWARE_TYPE_ERROR))
   }
 
+  const log = logger || _log
+  const logError = err => {
+    log('error', err)
+    if (err && err.sagaStack) {
+      log('error', err.sagaStack)
+    }
+  }
+
   const middleware = effectMiddlewares && compose(...effectMiddlewares)
 
-  const task = proc(
-    iterator,
+  const env = {
     channel,
```

Comment:
i would prefer naming this `stdChannel` as its not just any channel

Reply:
Yes, `channel` is too vague. How do you think naming this `defaultChannel` ?  

`stdChannel`  is already a function defined in *src/internal/channel.js*, and I am confused why it is a **standard channel**.

Reply:
`stdChannel` from `src/internal/channel.js` is channel factory. isn't it? dunno worth renaming it or not